test/smoke/clang-373823/Makefile

```diff
@@ -6,13 +6,13 @@ TESTSRC_AUX  =
 TESTSRC_ALL  = $(TESTSRC_MAIN) $(TESTSRC_AUX)
 RUNENV      += LIBOMPTARGET_KERNEL_TRACE=1
 
-CFLAGS       += -fopenmp-target-ignore-env-vars
```

Comment:
Is this flag not needed anymore? Seems to be missing on the other side. Is it because it is assumed by -fopenmp-target-fast?

Reply:
Yes, -fopenmp-target-fast includes the previous flag. With the old option, we don't get rpc:0 any more which is what the test is trying to check. I believe it is because of no-nested-par which target-fast implies.

Reply:
Sounds great! Thank you for clarifying that!